doc/getting-started.md

```diff
@@ -82,24 +93,65 @@ most simple capabilities is to add a delay.
 
 ```
 # First, create a rule file
-cat <<EOF > /tmp/hello-5s-rule.yaml
```

Comment:
How about we simply change this paragraph alone with % routing example and show how someone can test it?